.vscode/extensions.json

```diff
@@ -2,6 +2,7 @@
   "recommendations": [
     "EditorConfig.EditorConfig",
     "freebroccolo.reasonml",
+    "freebroccolo.input-assist",
```

Comment:
@jonsterling the input-assist plugin should be useful for typing unicode when we get around to implementing the pretty printer and stuff like that. It's basically a port of agda-mode.

Reply:
cool!